create-custom-map-style/src/transformRequest.js

```diff
@@ -0,0 +1,28 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Can this be replaced by Amplify Geo or amazon-location-helpers?

Reply:
Replacing it with `createMap` from `maplibre-gl-js-amplify`